components/Form/ScoreInput.tsx

```diff
@@ -0,0 +1,150 @@
+import { css, StyleSheet } from "aphrodite";
```

Comment:
hmm the conditionals-formatting looks off in this file. What kind of linter are you using?

Reply:
eslint. not sure why it skipped. Could be that it doesn't do space formatting?
I actually don't see spaces/tabs defined in eslint